sale_by_packaging/models/product_product.py

```diff
@@ -1,13 +1,35 @@
 # Copyright 2020 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import fields, models
+from odoo import api, fields, models
 from odoo.tools import float_compare, float_is_zero, float_round
 
 
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
+    min_sellable_qty = fields.Float(
+        compute="_compute_variant_min_sellable_qty",
+        readonly=True,
+        help=(
+            "Minimum sellable quantity, according to the available packagings, "
+            "if Only Sell by Packaging is set."
+        ),
+    )
+
+    @api.depends(
+        "sell_only_by_packaging", "uom_id.factor", "packaging_ids.qty",
+    )
+    def _compute_variant_min_sellable_qty(self):
+        for record in self:
+            record.min_sellable_qty = 0.0
+            if record.sell_only_by_packaging and record.packaging_ids:
+                record.min_sellable_qty = fields.first(
+                    record.packaging_ids.sorted(lambda p: p.qty)
+                ).qty
+            elif not record.sell_only_by_packaging:
+                record.min_sellable_qty = record.uom_id.factor
```

Comment:
Why this ?

Reply:
ping @mmequignon

Reply:
In my opinion, this `elif` should just be removed.
And also the `min_sellable_qty` field should only be displayed in the view if the product is set with `sell_only_by_packaging` set to True

Reply:
I implemented those changes on the forward port  to 14 #1784
I will bring them here if they are good.